content/en/blog/2023/ecs-otel-semconv-convergence.md

```diff
@@ -0,0 +1,50 @@
+---
+title:
+  Announcing the Elastic Common Schema (ECS) and OpenTelemetry Semantic
+  Convention Convergence
+linkTitle: ECS and OTel SemConv Convergence
+date: 2023-04-17
+spelling: cSpell:ignore ECS Reiley SemConv Yang
+author: '[Reiley Yang](https://github.com/reyang)'
+---
+
+Today, we're very excited to make a joint announcement with
+[Elastic](https://www.elastic.co/) about the future of
+[ECS](https://www.elastic.co/guide/en/ecs/master/ecs-reference.html) (Elastic
+Common Schema) and the OpenTelemetry Semantic Conventions.
+
+The goal is to achieve convergence of ECS and OTel Semantic Conventions into a
+single open schema that is maintained by OpenTelemetry, so that OpenTelemetry
+Semantic Conventions truly is a successor of the Elastic Common Schema.
+OpenTelemetry shares the same interest of improving the convergence of
+observability and security in this space. We believe this schema merge brings
+huge value to the open-source community because:
+
+- ECS has years of proven success in the logs, metrics, traces and security
+  events schema, providing great coverage of the common problem domains.
+- ECS provides schema for security domain fields, which is an important aspect
+  of telemetry.
+- Converging two separate standards into one single standard will help to boost
+  the ecosystem (e.g. instrumentation libraries, tools and consumption
+  experiences), which benefits both the telemetry producers and consumers.
+- This joint effort would benefit from domain experts across logging,
+  distributed tracing, metrics and security events. As a result, we expect to
+  have more consistent signals across different pillars of observability and
+  security events.
+
+Both Elastic and the OpenTelemetry community understand that converging two
+widely used standards into one singular common schema, and having a smooth
+transition is critical for users. A dedicated OpenTelemetry Semantic Convention
+working group will be created with domain experts from both Elastic and
+OpenTelemetry joining. We're also welcoming domain experts who are passionate
+about data schemas and semantic conventions to join us. If you're interested in
+contributing, join our
+[OTel Semantic Conventions working group](https://github.com/open-telemetry/community#specification-sigs),
+and join the discussion on our
+[Slack channel](https://cloud-native.slack.com/archives/C041APFBYQP).
+
+## References
+
+- [Announcement from Elastic](https://elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-announcement)
+- [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/concepts/semantic-conventions/)
+- [OTEP 199: Merge Elastic Common Schema with OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/oteps/blob/d02a3e2e75dc934fb38c5db88eb41fbe85730af4/text/0199-support-elastic-common-schema-in-opentelemetry.md)
```

Comment:
Note that the following edit fixes the build warning:

```
WARN 2023/04/12 11:26:49 blog/2023/ecs-otel-semconv-convergence.md: use a local path, not an external URL, for the following reference to a site local page: https://opentelemetry.io/docs/concepts/semantic-conventions/
```


```suggestion
- [Announcement](https://elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-announcement)
  from Elastic
- [OpenTelemetry Semantic Conventions][]
- [OTEP 199][]: _Merge Elastic Common Schema with OpenTelemetry Semantic
  Conventions_

[OpenTelemetry Semantic Conventions]: /docs/concepts/semantic-conventions/
[OTEP 199]:
  https://github.com/open-telemetry/oteps/blob/d02a3e2e75dc934fb38c5db88eb41fbe85730af4/text/0199-support-elastic-common-schema-in-opentelemetry.md

```

Reply:
@reyang - I see your thumbs-up, but the edits above (most importantly, the link fix to eliminate the warning), haven't been made.

Reply:
Yes @chalin, I pinged folks from Elastic and want them to confirm before I made these changes that are not purely editorial - just to make sure they don't see these as a last-minute surprise.

Reply:
Hi I don't understand the change from `Announcement from  Elastic` to just `Announcement` for the link?

Reply:
I've made editorial changes by reformatting the links, PTAL.

Reply:
I am ok either way - as long as our link is there. ;-) THANK YOU!!!